llama-index-integrations/llms/llama-index-llms-anthropic/llama_index/llms/anthropic/base.py

```diff
@@ -101,6 +102,15 @@ class Anthropic(FunctionCallingLLM):
     max_retries: int = Field(
         default=10, description="The maximum number of API retries.", gte=0
     )
+    vertexai_project_id: Optional[str] = Field(
+        default=None, description="Project ID for VertexAI in Google Cloud"
+    )
+    vertexai_region: Optional[str] = Field(
+        default=None, description="Region for VertexAI in Google Cloud"
+    )
+    vertexai_service_account_file: Optional[str] = Field(
```

Comment:
We have a VertexAI LLM class already, why do we need to add this here?

Reply:
We can't access anthropic models using that

Reply:
Then shouldn't the vertexai class be updated no? Or its actually impossible?

Reply:
To access anthropic models using vertexai we have to do it through anthropic SDK, which is already used in the llama-index anthropic class.